src/mongo/tree/MongoCollectionTreeItem.ts

```diff
@@ -173,8 +194,21 @@ export class MongoCollectionTreeItem implements IAzureParentTreeItem {
 			});
 	}
 
-	private insertMany(documents: any[]): Thenable<string> {
-		return this.collection.insertMany(documents)
+	private insertMany(args: any[]): Thenable<string> {
+		// documents = args[0], collectionWriteOptions from args[1]
+		let insertManyOptions: CollectionInsertManyOptions = {};
+		if (args.length > 2) {
+			throw new Error("Too many arguments. Please see mongo shell documentation. https://docs.mongodb.com/manual/reference/method/db.collection.insertMany/#db.collection.insertMany");
+		} else if (args.length === 2) {
+			if (args[1] && args[1].ordered) {
+				insertManyOptions["ordered"] = args[1].ordered;
+			}
+			if (args[1] && args[1].writeConcern) {
+				insertManyOptions["w"] = args[1].writeConcern;
+			}
+		}
```

Comment:
The sole-purpose of this is to rename the field _writeConcern_ to _w_

Reply:
We'll worry about improvements later